dpkg_requirements.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+DEPS = "python3-pip python3-opencv python3-setuptools python-execnet cmake libatlas-base-dev build-essential"
+apt-get update
+apt-get install -y $DEPS
```

Comment:
Remove `apt-get update`.
remove the variable and directly put it after `sudo apt install`.

And make it `sudo apt install` (use `sudo` and remove `-get`).

Also, add remaining deps (face-recognition and terminaltables).